extract/spacy_model.py

```diff
@@ -0,0 +1,101 @@
+import sys
+
+sys.path.append("../")
+import json
+import random
+import spacy
+import pandas as pd
+import os
+from lib.path import data_file_path
+
+# #  import labeled data that has been exported from doccano
+labeled_data = []
+
+with open(
+    data_file_path("raw/new_orleans_csc/appeal_hearing_pdfs/training_data/ayyub.jsonl")
```

Comment:
What tool did you use to create this file?

Reply:
This comes from Doccano right?

Reply:
Yes, this was jsonl is the output that I downloaded from Doccano after labeling my training set.